src/pudl/output/ferc714.py

```diff
@@ -196,455 +194,437 @@ def categorize_eia_code(eia_codes, ba_ids, util_ids, priority="balancing_authori
-
-    def georef_counties(self, update=False):
-        """Annual respondents with all associated county-level geometries.
-
-        Given the county FIPS codes associated with each respondent in each year, pull
-        in associated geometries from the US Census DP1 dataset, so we can do spatial
-        analyses. This keeps each county record independent -- so there will be many
-        records for each respondent in each year. This is fast, and still good for
-        mapping, and retains all of the FIPS IDs so you can also still do ID based
-        analyses.
-        """
-        if update or self._counties_gdf is None:
-            census_counties = pudl.etl.defs.load_asset_value(
-                AssetKey("county_censusdp1")
-            )
-            self._counties_gdf = pudl.analysis.service_territory.add_geometries(
-                self.fipsify(update=update), census_gdf=census_counties
-            ).pipe(apply_pudl_dtypes)
-        return self._counties_gdf
-
-    def georef_respondents(self, update=False):
-        """Annual respondents with a single all-encompassing geometry for each year.
-
-        Given the county FIPS codes associated with each responent in each year, compile
-        a geometry for the respondent's entire service territory annually. This results
-        in just a single record per respondent per year, but is computationally
-        expensive and you lose the information about what all counties are associated
-        with the respondent in that year. But it's useful for merging in other annual
-        data like total demand, so you can see which respondent-years have both reported
-        demand and decent geometries, calculate their areas to see if something changed
-        from year to year, etc.
-        """
-        if update or self._respondents_gdf is None:
-            census_counties = pudl.etl.defs.load_asset_value(
-                AssetKey("county_censusdp1")
-            )
-            self._respondents_gdf = (
-                pudl.analysis.service_territory.add_geometries(
-                    self.fipsify(update=update),
-                    census_gdf=census_counties,
-                    dissolve=True,
-                    dissolve_by=["report_date", "respondent_id_ferc714"],
-                )
-                .merge(
-                    self.summarize_demand(update=update)[
-                        ["report_date", "respondent_id_ferc714", "demand_annual_mwh"]
-                    ]
-                )
-                .pipe(apply_pudl_dtypes)
+            tables.append(table)
+            if "replace" in util and util["replace"]:
+                mask |= is_child
+    return pd.concat([df[~mask], pd.concat(tables)]).drop_duplicates()
+
+
+@asset(compute_kind="Python")
+def filled_service_territory_eia861(
+    filled_balancing_authority_assn_eia861: pd.DataFrame,
+    service_territory_eia861: pd.DataFrame,
+) -> pd.DataFrame:
+    """Modified service_territory_eia861 table."""
```

Comment:
The doc strings for some of these intermediate norm_ferc714 tables aren't super verbose. What is this function doing?